src/meson.build

```
@@ -98,7 +98,7 @@ graphene_dep_sources = []
 # Introspection
 if build_gir
   python = python.find_installation('python3')
-  identfilter_py = join_paths(meson.current_source_dir(), 'identfilter.py')
+  identfilter_py = '"' + join_paths(meson.current_source_dir(), 'identfilter.py') + '"'
 
   gir_extra_args = [
     '--identifier-filter-cmd=@0@ @1@'.format(python.full_path(), identfilter_py),
```

Does it work if you put the quotes here, instead:

```suggestion
    '--identifier-filter-cmd="@0@ @1@"'.format(python.full_path(), identfilter_py),
```

---

`"@0@ @1@"` doesn't work, but `@0@ "@1@"` does. I've updated the patch.